src/main/java/redis/clients/jedis/Jedis.java

```diff
@@ -108,8 +108,8 @@ public String get(final String key) {
   }
 
   /**
-   * Test if the specified key exists. The command returns the number of keys existed
-   * Time complexity: O(N)
+   * Test if the specified key exists. The command returns the number of keys existed Time
```

Comment:
I didn't format HostAndPort before checking it in.  However if we're going to change the formatter, it's much easier to read (at least for me) with 4 spaces instead of two.  When it gets very nested, its more difficult to to, at least for me.